models/topics.models.js

```diff
@@ -1,8 +1,15 @@
 const db = require("../db/connection");
-const format = require("pg-format");
 
 exports.selectTopics = () => {
   return db.query("SELECT * FROM topics").then((topics) => {
     return topics.rows;
   });
 };
+
+exports.selectArticlesById = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then((articles) => {
+      return articles;
```

Comment:
You could take the articles off the key of rows here, otherwise there's not much point in this `.then()` block